app/controllers/chat_controller.rb

```diff
@@ -498,9 +498,9 @@ def find_chatable
   end
 
   def find_chat_message
-    @message =
-      ChatMessage.unscoped.includes(chat_channel: :chatable).find_by(id: params[:message_id])
-
+    @message = ChatMessage.unscoped.includes(chat_channel: :chatable)
```

Comment:
Can the `unscoped` be abused in any ways? Usually, only staff/admins can query stuff unscoped. Is the `find_chat_message` properly protected?